spanner/ddl/ast_test.go

```diff
@@ -102,15 +101,43 @@ func TestPrintCreateTable(t *testing.T) {
 	tests := []struct {
 		name       string
 		protectIds bool
-		expected   string
 		ct         CreateTable
+		expected   string
 	}{
-		{"no quote", false, "CREATE TABLE mytable (col1 INT64 NOT NULL, col2 STRING(MAX), col3 BYTES(42)) PRIMARY KEY (col1 DESC)", t1},
-		{"quote", true, "CREATE TABLE `mytable` (`col1` INT64 NOT NULL, `col2` STRING(MAX), `col3` BYTES(42)) PRIMARY KEY (`col1` DESC)", t1},
-		{"interleaved", false, "CREATE TABLE mytable (col1 INT64 NOT NULL, col2 STRING(MAX), col3 BYTES(42)) PRIMARY KEY (col1 DESC),\nINTERLEAVE IN PARENT parent", t2},
+		{
+			"no quote",
+			false,
+			t1,
+`CREATE TABLE mytable (
+  col1 INT64 NOT NULL,
+  col2 STRING(MAX),
+  col3 BYTES(42),
+) PRIMARY KEY (col1 DESC)`,
+		},
+		{
+			"quote",
+			true,
+			t1,
+"CREATE TABLE `mytable` (\n"+
```

Comment:
For consistency, suggest using ` string here as well?

Reply:
gofmt formatted ` ... ` strings as following:

			`CREATE TABLE mytable (
  col1 INT64 NOT NULL,
  col2 STRING(MAX),
  col3 BYTES(42),
) PRIMARY KEY (col1 DESC)`,

For consistency, have modified all strings to use " ... " concatanated with newlines.